server/store/string_store.go

```diff
@@ -9,6 +9,9 @@ import "fmt"
 // PrefixInfohash is the prefix to be used for infohashes.
 const PrefixInfohash = "ih-"
 
+// PrefixPeerID is the prefix to be used for client peer IDs.
+const PrefixPeerID = "pi-"
```

Comment:
I'd call this `PrefixClientID`. And I don't know about the value of the prefix, could just be `c-` if you ask me, I chose `ih-` for infohashes because it kinda made sense, but afaik we don't have rules for the prefixes.
@jzelinskie  ?